Documentation/planning/arcade-powered-source-build/README.md

```diff
@@ -0,0 +1,155 @@
+# Arcade-powered source-build
+
+Source-build existing outside the Microsoft build process impedes efforts to
+maintain the source-buildability of .NET Core. Integration into the normal build
+process is driven by two major goals:
+
+1. Official build. When the .NET Core SDK official build completes, its
+   artifacts include validated, ship-ready source-build outputs, in addition to
+   the Microsoft build outputs.
+
+   * This has two core benefits over the current situation: we know immediately
+     whether an SDK can be source-built cleanly, and the status is as visible as
+     an SDK build failure.
+
+   * Breaks in source-build can be fixed immediately by repo owners, just as any
+     other build failure, rather than fixed by source-build maintainers days
```

Comment:
I think "days after" is optimistic.  It's usually weeks, because we need to wait for a coherent build before even attempting to build.

Reply:
Ah, I was originally thinking days since we get a coherent build, but that's not what I'm writing about here really... changing to "weeks to months", the worst case being someone making a change in `master`--since we have to wait for a *very* long time for those to bubble up.